src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -195,15 +195,15 @@ export default class AircraftController {
             // let is_visible = aircraft_visible(aircraft);
 
             if (aircraft.isStopped() && aircraft.category === 'arrival') {
-                window.gameController.game.score.windy_landing += aircraft.scoreWind('landed');
+                aircraft.scoreWind('landed');
 
                 window.uiController.ui_log(`${aircraft.getCallsign()} switching to ground, good day`);
                 speech_say([
                     { type: 'callsign', content: aircraft },
                     { type: 'text', content: ', switching to ground, good day' }
                 ]);
 
-                window.gameController.game.score.arrival += 1;
+                window.gameController.events_recordNew('ARRIVAL');
```

Comment:
make sure to update these to use the new enumeration.  This should now be `GAME_EVENTS.ARRIVAL`.